tests/suites/os/tests/udev/index.js

```diff
@@ -66,6 +66,18 @@ module.exports = {
 					'All required by-state links have been created'
 				);
 			}
-		}
+		},
+		{
+			title: 'rootfs by-state links are unique',
+			run: async function(test) {
+				return test.resolves(
+					this.worker.executeCommandInHostOS(
+						`test /dev/disk/by-state/active != /dev/disk/by-state/inactive`,
```

Comment:
We were never able to reproduce this with QEMU RAID were we?

Reply:
No, I never was able to trigger the same bug in qemu